src/librustdoc/html/static/js/main.js

```diff
@@ -235,13 +242,12 @@ function preLoadCss(cssUrl) {
         document.head.append(script);
     }
 
-    if (getSettingsButton()) {
-        // @ts-expect-error
-        getSettingsButton().onclick = event => {
+    const settingsButton = getSettingsButton();
+    if (settingsButton) {
```

Comment:
In the unlikely event that the settings button cannot be located, I think it would be better to fail-fast, rather than silently ignore it, no?

Reply:
Fair, I suppose.